packages/cli/src/utils/pathToChange.ts

```diff
@@ -0,0 +1,23 @@
+
+import { resolve as resolvePath, sep } from 'path'
+import { Readable } from 'stream'
+import { readFileSync } from 'fs-extra'
+import { getRoot } from './directory'
+
+export interface ChangeToCopy {
```

Comment:
Instead of `change`, how about using `update`? I think `change` is a bit too generic, and the names `ChangeToCopy` and `pathToChange` don't tell me much about the actual use of both the interface and the function

Reply:
Awesome! I will change those names.